app/build.gradle

```diff
@@ -124,6 +124,8 @@ dependencies {
     implementation 'com.google.guava:guava:28.1-jre'
     annotationProcessor 'com.google.guava:guava:24.1-jre'
     implementation 'org.apache.commons:commons-lang3:3.9'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
Here you may want to prefer to use androidx instead of android support which is deprecated.

Here is the right one: `implementation 'androidx.constraintlayout:constraintlayout:1.1.3'`